pvlib/iam.py

```diff
@@ -940,3 +942,368 @@ def schlick_diffuse(surface_tilt):
         cug = pd.Series(cug, surface_tilt.index)
 
     return cuk, cug
+
+
+def _get_model(model_name):
+    # check that model is implemented
+    model_dict = {'ashrae': ashrae, 'martin_ruiz': martin_ruiz,
+                  'physical': physical}
+    try:
+        model = model_dict[model_name]
+    except KeyError:
+        raise NotImplementedError(f"The {model_name} model has not been \
+                                    implemented")
```

Comment:
```suggestion
        raise NotImplementedError(f"The {model_name} model has not been "
                                  "implemented")
```

The backslash approach results in a bunch of whitespace inside the error message string.